govcd/vapptemplate.go

```diff
@@ -123,3 +125,16 @@ func (vAppTemplate *VAppTemplate) UpdateAsync() (Task, error) {
 	return vAppTemplate.client.ExecuteTaskRequest(url, http.MethodPut,
 		types.MimeVAppTemplate, "error updating vApp template item: %s", vappTemplatePayload)
 }
+
+// Delete deletes the VAppTemplate Item, returning an error if the vCD call fails.
```

Comment:
Nit. The `vCD` comes from the vCloud Director era. We're not VMware Cloud Director :)

```suggestion
// Delete deletes the VAppTemplate Item, returning an error if the VCD call fails.
```

Reply:
Changed, thanks for spotting it!